frontend/src/components/pages/CreatePasswordPage.tsx

```diff
@@ -0,0 +1,202 @@
+import React from "react";
+import {
+  Input,
+  Button,
+  Center,
+  Stack,
+  Flex,
+  Box,
+  Text,
+  FormLabel,
+  FormControl,
+} from "@chakra-ui/react";
+import ResponsiveLogo from "../common/responsive/ResponsiveLogo";
+import ResponsivePasswordInput from "../common/responsive/ResponsivePasswordInput";
+import ResponsiveAuthContainer from "../common/responsive/ResponsiveAuthContainer";
+import ResponsiveModalWindow from "../common/responsive/ResponsiveModalWindow";
+import background from "../assets/background.png";
+import backgroundMobile from "../assets/background_mobile.png";
+
+const CreatePasswordPage = (): React.ReactElement => {
+  const [showModal, setShowModal] = React.useState(false);
+  const [password, setPassword] = React.useState("");
+  const [confirmPassword, setConfirmPassword] = React.useState("");
+  const [errorMessage, setErrorMessage] = React.useState("");
+
+  const handlePasswordChange = (event: React.ChangeEvent<HTMLInputElement>) => {
+    setPassword(event.target.value);
+  };
+  const handleConfirmPasswordChange = (
+    event: React.ChangeEvent<HTMLInputElement>,
+  ) => {
+    setConfirmPassword(event.target.value);
+  };
+
+  const validatePasswords = () => {
+    setErrorMessage("");
+    if (password.length < 8) {
+      setErrorMessage("Password must be at least 8 characters.");
+      return true;
```

Comment:
lowkey this is a bit counter intuitive. For better readability can you make this function return true if the passwords are valid, and false if the passwords are invalid?